packages/expo-dev-menu/android/src/debug/java/expo/modules/devmenu/DevMenuDevSettings.kt

```diff
@@ -22,6 +23,15 @@ object DevMenuDevSettings {
         putBoolean("isElementInspectorAvailable", devSettings.isJSDevModeEnabled)
         putBoolean("isHotLoadingAvailable", devSettings.isJSDevModeEnabled)
         putBoolean("isPerfMonitorAvailable", devSettings.isJSDevModeEnabled)
+        putBoolean("isJSInspectorAvailable", run {
+          val packageName = reactInstanceManager.currentReactContext?.packageName
+            ?: return@run false
+          val metroHost = "http://${devSettings.packagerConnectionSettings.debugServerHost}"
+          runBlocking {
+            DevMenuManager.metroClient
+              .queryJSInspectorAvailability(metroHost, packageName)
+          }
+        })
```

Comment:
It seems potentially problematic to run this blocking query synchronously when creating the dev menu activity. Could we put this in an async function for JS to query, instead of delaying the activity / delegate being created?